### ISSUE TITLE
Create slack like service for TechnoCore

### ISSUE THREAD

**Issue by TheHackmeister** [SciFiFarms/TechnoCore#29]
There are a number of possible slack alternatives in my [software to try list](https://github.com/SciFiFarms/TechnoCore-Docs/blob/master/pages/allthing/allthing_software_to_try.md). However, I think there are a number of projects that user gitter. So maybe a little more research is needed as to why that is.

**Comment by bengroneman** [comment]
Mattermost seems intriguing. I have a DO server, do you mind if I take a crack at this?

**Comment by TheHackmeister** [comment]
Yeah, that would be sweet! Let me know if there is anything I can do to help.

**Comment by bengroneman** [comment]
Great!

I am using a DigitalOcean server and have an instance running [here](http://159.89.159.135/scififarms/).

I've invited you with a link through email. 

There are a few further steps that need to be taken for this to be finalized.

- Use a domain name. What would you want this to be? (B + S)
- Use TLS leveraging [certbot](https://certbot.eff.org/lets-encrypt/ubuntubionic-other) (B)
- Setup the configuration for [SMTP](https://docs.mattermost.com/install/smtp-email-setup.html) (B + S)

**Comment by TheHackmeister** [comment]
Great! I pointed chat.scifi.farm to that server, so you should be able to use certbot to verify that address. I also set up an SMTP account and sent the credentials to you on the chat server. If you run into any trouble getting it set up, let me know :)

Thanks for setting it up! Getting chat is bad ass!

**Comment by TheHackmeister** [comment]
Also, if you want help funding the server, let me know. I'm running a couple of other servers, so it's not a big deal for me to cover it.
